tests/acceptance/features/apiGraph/fullSearch.feature

```diff
@@ -88,6 +88,29 @@ Feature: full text search
       | spaces           |
 
 
+  Scenario Outline: search project space folders by tag
+    Given using spaces DAV path
+    And the administrator has assigned the role "Space Admin" to user "Alice" using the Graph API
+    And user "Alice" has created a space "tag-space" with the default quota using the GraphApi
+    And user "Alice" has created a folder "spacesFolder/spacesSubFolder" in space "tag-space"
+    And user "Alice" has created a folder "unTaggedSpacesFolder/unTaggedSpacesSubFolder" in space "tag-space"
+    And user "Alice" has created the following tags for folder "spacesFolder" of the space "tag-space":
+      | tag1 |
+    And user "Alice" has created the following tags for folder "spacesFolder/spacesSubFolder" of the space "tag-space":
+      | tag1 |
+    And using <dav-path-version> DAV path
+    When user "Alice" searches for "Tags:tag1" using the WebDAV API
+    Then the HTTP status code should be "207"
+    And the search result of user "Alice" should contain only these files:
+      | spacesFolder    |
+      | spacesSubFolder |
+    Examples:
+      | dav-path-version |
+      | old              |
+      | new              |
+      | spaces           |
```

Comment:
```suggestion
```
I think make a single scenario since its about project space. There is no way we can create space project with `new` and `old` webdav and on the top of that you are using `Given using spaces DAV path` at the beginning.

Reply:
`Given using spaces DAV path` this step will help in making the files in project spaces
And below step will search the project space files in different endpoints
```
Given using <dav-path-version> path
Examples:
      | dav-path-version |
      | old              |
      | new              |
      | spaces           |
```